bpf/kmesh/workload/include/config.h

```diff
@@ -15,21 +15,21 @@
 #define MAP_SIZE_OF_AUTH_POLICY   512
 
 // rename map to avoid truncation when name length exceeds BPF_OBJ_NAME_LEN = 16
-#define map_of_frontend       km_frontend
-#define map_of_service        km_service
-#define map_of_endpoint       km_endpoint
-#define map_of_backend        km_backend
-#define map_of_auth           km_auth
-#define map_of_tuple          km_tuple
-#define map_of_tcp_info       km_tcpinfo
-#define map_of_authz          km_authz
-#define map_of_dst_info       km_dstinfo
-#define map_of_tail_call_prog km_tailcallprog
-#define xdp_tailcall_map      km_xdptailcall
```

Comment:
please take a look at this file only, other modifications are all to adapt to this

Reply:
I think this is a good modification. 
But it would also be nice to add comments to some of the maps with more detail. Such as `km_cgr_tailcall`.
After all, names are limited in length.

Reply:
I do think comments at the place where map defined is better, because `xdp_tailcall_map` is used every where